modules/account-lib/src/coin/eth/utils.ts

```diff
@@ -270,6 +286,92 @@ export function decodeTokenTransferData(data: string): TokenTransferData {
   };
 }
 
+export function decodeERC721TransferData(data: string): ERC721TransferData {
+  if (!data.startsWith(sendMultisigMethodId)) {
+    throw new BuildTransactionError(`Invalid transfer bytecode: ${data}`);
+  }
+
+  const [to, amount, internalData, expireTime, sequenceId, signature] = getRawDecoded(
+    sendMultiSigTypes,
+    getBufferedByteCode(sendMultisigMethodId, data),
+  );
+
+  // FIXME: Is this corrrect?
```

Comment:
Was this resolved with other commits or is the FIXME still active?

Reply:
No this was resolved. Thanks for pointing it out